draft-ietf-capport-architecture.xml

```diff
@@ -256,6 +255,14 @@
         portal web application, and (c) legacy devices must continue to be
         supported.
         </t>
+        <t>
+        If User Equipment supports Captive Portal API, it MUST use TLS to communicate with
+        Captive Portal API and MUST validate API server TLS certificate (see <xref target="RFC2818"/>).
+        Enforcement device SHOULD allow User Equipment to connections to OCSP responders, CRLs and
+        NTP servers while still in captive mode (see section 4.1 in
+        <xref target="I-D.ietf-capport-api"/>) to validate the server certificate. If certificate
```

Comment:
Do you think it may be a mistake to reference the API doc here? It forms a circular dependency between them. I'm not sure what the IETF's view on that is, but it seems like it's not the greatest. Maybe just the text you have without the reference sufficient? It seems pretty obvious why it's necessary.

Reply:
An informative reference is fine.  These will ship together, so a loop isn't fatal.

Reply:
moved the reference to informative section